### PR TITLE
Handling invitations after guest accepts and/or rejects invitations

### PR DESCRIPTION
**Pull request by chanie01234**
## Description
Once an invitation is sent, the user has the option of accepting or rejecting. If rejected, the invite is removed from pending invite list. If accepted, the invite appears on the user dashboard where you can see more detailed info on the trip. 

## Related Issues
https://caravanning.atlassian.net/browse/CA-140

## Testing
Add a step-by-step list on how to test this PR, like this:
- [ ] Run `node server.js`
- [ ] Open Postman
- [ ] Enter DELETE `localhost:8080/invitations/:userid/:tripid`
## How are you feeling?
👌

### CODE REVIEW

**Comment by chanie01234** [comment]
I don't know what the hell I'm doing smh sorry guys. I'll revisit this again as well as the other PR about the trips

File: server.js [hunk 1]
```diff
@@ -116,6 +116,25 @@ app.get('/invitations/:userid', (req, res) => {
 	);
 });
 
+app.delete('/invitations/:userid/:tripid', (req, res) => {
+	pool.query(
```
Comment (kcode20) [inline server.js:120]: I can give you help in the right direction. Here are a few steps I think you should follow:
1. Start by console.log-ing `req.params`: 
```jsx
console.log(req.params)
```
`req.param` is the parameters that are on the request object that we send to the server. In the URL, you attached two parameters `userid` and `tripid`. 
2. Run the server (running `node server.js`) 
3. Go to Postman and run `DELETE` of `/invitations/:userid/:tripid`
Screen Shot 2020-04-20 at 12 21 46 PM
4. Go to the terminal you ran `node server.js` now you should see the value of `userid` and `tripid` on the req.params. 
5. Now add to your query the value of not only `req.params.userid` but also `req.params.tripid`.

**Comment by chanie01234** [comment]
So look at what I did:
Screen Shot 2020-04-27 at 3 15 43 AM

I inputted `localhost:8080/invitations/1/2` where `1` is the `userid` and `2` is the `tripid` and got the response `OK`  but nothing happens in the database on Postico

**Comment by chanie01234** [comment]
I also noticed that if `accepted` state is `TRUE`, the API call is `Forbidden`

**Pull request merged**